mlflow/tracking/client.py

```diff
@@ -2281,7 +2281,7 @@ def log_image(
             image = image.to_pil()
         else:
             # Import PIL and check if the image is a PIL image
-            import PIL
+            import PIL.Image
```

Comment:
how did this line pass tests when it was initially added?

Reply:
or it's not tested at all?

Reply:
It passes because we create PIL images in the test, therefore they're in scope during the test (see line 37 for an example in `tests/tracking/test_log_image.py` below)

Reply:
Python amazes me every day